rotkehlchen/exchanges/binance.py

```diff
@@ -589,6 +607,184 @@ def _query_lending_balances(
+                                location='binance lending interest processing',
+                            ).to_int(exact=False),
+                        )
+                        asset = asset_from_binance(entry['asset'])
+                    except (UnsupportedAsset, UnknownAsset) as e:
+                        error_type = 'unknown' if isinstance(e, UnknownAsset) else 'unsupported'
+                        self.msg_aggregator.add_warning(
+                            f'Found {error_type} {self.name} asset {e.identifier}. '
+                            f'Ignoring its lending interest history query.',
+                        )
+                        continue
+                    except KeyError as e:
+                        self.msg_aggregator.add_error(
+                            f'Missing key entry for {str(e)} in {self.name} {entry}. '
+                            f'Ignoring its lending interest history query.',
+                        )
+                        continue
+                    except DeserializationError as e:
+                        self.msg_aggregator.add_error(
+                            f'Error at deserializing {self.name} asset. {str(e)}. '
+                            f'Ignoring its lending interest history query.',
+                        )
+                        continue
+
+                    event = HistoryBaseEntry(
+                        event_identifier=event_identifier,
+                        sequence_index=idx,
+                        timestamp=timestamp,
+                        location=self.location,
+                        location_label=self.name,
+                        asset=asset,
+                        balance=Balance(
+                            amount=interest_received,
+                            usd_value=ZERO,
+                        ),
+                        notes=f'Interest paid from {entry["lendingType"]} {entry["productName"]} savings',  # noqa: E501
+                        event_type=HistoryEventType.RECEIVE,
+                        event_subtype=HistoryEventSubType.REWARD,
+                    )
+                    events.append(event)
+
+            if len(events) != 0:
+                with self.db.user_write() as write_cursor:
+                    try:
+                        self.history_events_db.add_history_events(write_cursor, events)
+                    except InputError as e:
+                        self.msg_aggregator.add_error(
+                            f'Failed to save Binance lending interest history {query_start_ts} '
+                            f'to {query_end_ts} in database. {str(e)}',
+                        )
+
+            with self.db.user_write() as write_cursor:
+                ranges.update_used_query_range(
+                    write_cursor=write_cursor,
+                    location_string=range_query_name,
+                    queried_ranges=[(start_ts, end_ts)] + ranges_to_query,
```

Comment:
Question about this. I've seen that we to the same in other places but isn't this equivalent to 

```
queried_ranges=[(start_ts, end_ts)] 
```
? What is the difference?

Reply:
I think I shouldn't have included the `ranges_to_query` since I've not queried it.